### ISSUE TITLE
Make prefixes for names, e.g. ELB name, configurable.

### ISSUE THREAD

**Issue by DanielMaier-BSI** [docker/for-aws#13]
### Expected behavior
It would be nice to have an input parameter to define the prefix for all the names, e.g. ELB name, ag name etc., as default AWS::StackName can be used.

### Actual behavior
AWS::StackName gets used as prefix which leads to too long names (max. is 32 for example for ELB name) when you use docker for aws template as nested stack.

### Steps to reproduce the behavior

  1. Use docker for aws template as nested stack. Example:
```
AWSTemplateFormatVersion: "2010-09-09"
Parameters:
    ClusterSize: 
        Default: '5'
        Description: 'Number of worker nodes in the Swarm (0-1000).'
        MaxValue: '1000'
        MinValue: '0'
        Type: Number
    KeyName: 
        ConstraintDescription: 'must be the name of an existing EC2 KeyPair.'
        Description: 'Name of an existing EC2 KeyPair to enable SSH access to the instances'
        MinLength: '1'
        Type: 'AWS::EC2::KeyPair::KeyName'
Resources:
    DockerSwarm:
        Type: "AWS::CloudFormation::Stack"
        Properties:
            TemplateURL: https://editions-us-east-1.s3.amazonaws.com/aws/stable/Docker.tmpl
            Parameters:
                KeyName: !Ref KeyName
                ClusterSize: !Ref ClusterSize
                ManagerSize: 3
                EnableSystemPrune: 'yes'
                EnableCloudWatchLogs: 'yes'
```
  2. Start stack and wait for CREATE_FAILED event.

**Comment by kencochrane** [comment]
The ELB name will be removed in the next Edge release, this will remove the issue where you can only have 32 characters. This will hopefully solve the issue you are seeing, without needing to add another parameter.

**Comment by kencochrane** [comment]
the latest edge release (17.04 CE) has ELB name removed, this will be available in the 17.06 stable release. closing this issue since it is now available on edge.
